### ISSUE TITLE
dynamic: default render group by process?

### ISSUE THREAD

**Issue by mr-tz** [mandiant/capa#1866]
for verbose output we see the process name/details

I think for the dynamic context this could be helpful to show in the default output as well

related: show a summary/tree view of the traced processes

as test example use (UPX packed): cd2cba9e6313e8df2c1273593e649682 which creates a service resulting in execution in `services.exe` and `svchost.exe`

**Comment by mr-tz** [comment]
I also think it makes more sense to show the capabilities in sequential/execution order. This tells analysts much more about the events and their context.

**Comment by williballenthin** [comment]
this is quite neat!

i think we should create some mockups or sketches to help illustrate the various designs and fine tune things before we implement them.

i also dont think this necessarily blocks the dynamic PR merge, though it would be nice to implement before long. do you agree?

**Comment by mr-tz** [comment]
Yes, all the recent issues are not blockers in my opinion. Main goal this week should be to get the PR merged.
